alacritty/src/event.rs

```diff
@@ -381,16 +384,37 @@ impl<N: Notify + OnResize> Processor<N> {
             config,
             message_buffer,
             display,
+            event_queue: Vec::new(),
+        }
+    }
+
+    #[inline]
+    /// Return `true` if `event_queue` is empty, `true` otherwise.
```

Comment:
There's a much more performant way to implement what this comment suggests: `return true`.

```suggestion
    /// Return `true` if `event_queue` is empty, `false` otherwise.
```

That might be more appropriate?